Test.C

```diff
@@ -471,20 +529,21 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
     printf("|------------------------------ Encryption Store ------------------------------|\n");
     printf("You currently have %i credits\n", playerPtr->currentCredits);
 
-    printf(" # | Encryption Name      | Cost  | Current Effectiveness\n");
-    printf("---|----------------------|-------|----------------------\n");
+    printf(" # | Encryption Name                        | Cost  | Current Effectiveness\n");
+    printf("---|----------------------------------------|-------|----------------------\n");
     for (int i=0; i<NUM_ENCRYPTION; i++){
-        printf(" %d | %20s | %5d | %.2f\n", i + 1, encryptionStock[i].name, encryptionStock[i].cost, encryptionStock[i].effectiveness);
+        printf(" %d | %40s | %5d | %.2f\n", i + 1, encryptionStock[i].name, encryptionStock[i].cost, encryptionStock[i].effectiveness);
     }
 
     printf("\n");
-    printf("Please select the number of the encryption you would like to buy: ");
+    printf("Please select the number of the encryption view more details: ");
     scanf("%d", &encryptSelect);
     // adjust encryption selection by one, since user input starts at one and not zero
     encryptSelect -= 1;
 
     if (encryptSelect >= 0 && encryptSelect < (NUM_ENCRYPTION)){
         printf("You have selected: %s\n", encryptionStock[encryptSelect].name);
+        printf("%s\n", encryptionStock[encryptSelect].name);
```

Comment:
I think you meant the description here.
```suggestion
        printf("%s\n", encryptionStock[encryptSelect].description);
```